spec.html

```diff
@@ -119,9 +119,9 @@ <h1>AggregateError ( _errors_, _message_ )</h1>
         <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
         <emu-alg>
           1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
-          1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]], [[AggregateErrors]] »).
+          1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
           1. Let _errorsList_ be ? IterableToList(_errors_).
-          1. Set _O_.[[AggregateErrors]] to _errorsList_.
+          1. Perform ! CreateDataPropertyOrThrow(_O_, `"errors"`, _errorsList_).
```

Comment:
I think this should be moved down. I know there's discussion about this in https://github.com/tc39/proposal-promise-any/pull/59, but this PR adds an additional reason, which is:

- Below we say "Let \_error\_ be a newly created \`AggregateError\` object; Perform ! CreateDataPropertyOrThrow(\_error\_, \`"errors"\`, \_errors\_)", which means
- The "errors" property on AggregateError instances created by `Promise.any` is added after the "message" property, which is observable with `Reflect.ownKeys` etc, which means
- The "errors" property on AggregateError instances created by `new AggregateError` should be too.

Reply:
Good catch. Agreed that the property creation order should match between the two scenarios.

Reply:
I agree. The observable part related to argument order is `IterableToList(errors)`, and that happens before the ToString of `message`; but the property creation order should match regardless. Fixed.